# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/libavutil/fifo.h
# Commit: 186e2a334f2d
# Full Hash: 186e2a334f2deb37b29f44b28bc8c83babeeb9a4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-09 09:37:17
# Regressor Bug: 1877221
# File Overlap Count: 1
# Description:
#   Bug 1877221 - Pull in new ffmpeg code, reapply patch. r=media-playback-reviewers,chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D199909
# ==============================================================================

diff -r c5920de78d16 -r 186e2a334f2d media/ffvpx/libavutil/fifo.h
--- a/media/ffvpx/libavutil/fifo.h	Thu Feb 08 10:39:53 2024 +0000
+++ b/media/ffvpx/libavutil/fifo.h	Thu Feb 08 10:39:54 2024 +0000
@@ -200,7 +200,7 @@
  *
  * @return a non-negative number on success, a negative error code on failure
  */
-int av_fifo_peek(AVFifo *f, void *buf, size_t nb_elems, size_t offset);
+int av_fifo_peek(const AVFifo *f, void *buf, size_t nb_elems, size_t offset);
 
 /**
  * Feed data from a FIFO into a user-provided callback.
@@ -217,7 +217,7 @@
  *
  * @return a non-negative number on success, a negative error code on failure
  */
-int av_fifo_peek_to_cb(AVFifo *f, AVFifoCB write_cb, void *opaque,
+int av_fifo_peek_to_cb(const AVFifo *f, AVFifoCB write_cb, void *opaque,
                        size_t *nb_elems, size_t offset);
 
 /**